modules/boot/src/actor_proc_data.cpp

```diff
@@ -20,4 +20,9 @@ KEEP_FUNC void loadActorName() {
         int offset = (g_currentActor->mBase.mProcName * 32);
         loadFile("twwgz/procs.bin", &g_procData, sizeof(g_procData), offset);
     }
+}
+
+KEEP_FUNC void loadProcInit(int& i_procName) {
+    int offset = i_procName * 32;
+    loadFile("twwgz/procsInit.bin", &g_procInit, sizeof(g_procInit), offset);
```

Comment:
I would name these after "player proc" or "player actions" so its more clear what these are, but not a big deal